src/components/LearnMoreSection/LearnMoreSection.css

```diff
@@ -0,0 +1,61 @@
+.learn-more-section {
+  display: flex;
+  align-items: center;
+  gap: 100px;
+}
+.learn-more-section .learn-more-image {
+  width: 660px;
+  height: 540px;
+}
```

Comment:
- [ ] Remove the width and height and apply 100% width to the image.

Reply:
Fixed